src/Lumi/Components2/Text.purs

```diff
@@ -0,0 +1,194 @@
+module Lumi.Components2.Text where
```

Comment:
We might not want to expose `mkHeaderComponent` just for consistency. Do you think the `Text` constructor should be exposed (don't know, just curious)?

Reply:
Ah yes, I forgot to constrain the exports :sweat_smile: